python/pyspark/sql/functions/builtin.py

```diff
@@ -4564,7 +4585,7 @@ def array_agg(col: "ColumnOrName") -> Column:
 
 
 @_try_remote_functions
-def collect_set(col: "ColumnOrName") -> Column:
+def collect_set(col: "ColumnOrName", ignorenulls: Optional[bool] = None) -> Column:
```

Comment:
can we match it to `ignoreNulls: Optional[Union[bool, Column]] = None`?

Reply:
also let's match the name to `ignoreNulls` with Scala side.

Reply:
What is the correct what to make that work. Add an overload on the scala side that also takes `Column` for ignoreNulls. Or change the python side to call the method using `Column.fn` directly?

Reply:
I think Scala should also take Column, and fix Python to take both